biz.aQute.bndlib/src/aQute/bnd/build/model/EE.java

```diff
@@ -145,6 +145,17 @@ public Version getCapabilityVersion() {
 		return capabilityVersion;
 	}
 
+	/**
+	 * @return the java release number for this EE
+	 */
+	public int getRelease() {
+		int major = getCapabilityVersion().getMajor();
+		if (major > 8) {
+			return major;
+		}
+		return 0;
```

Comment:
Since this value is not useful, perhaps the method should return an `OptionalInt` where being empty correctly represents no release target value.

Reply:
I would prefer not to make this change as it forces the user of the EE (e.g. the resolver) to check for empty every time rather than just being able to ask the Jar for “Manifest at releaseTarget” and not worry about whether the EE is 1.8.

Reply:
Done.